File: CS/WinWebSolution.Module.Win/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;

namespace WinWebSolution.Module.Win {
    public class Updater : ModuleUpdater {
        public Updater(DevExpress.ExpressApp.IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
        }
    }
}

File: VB/WinWebSolution.Module.Win/Updater.vb
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo

Namespace WinWebSolution.Module.Win
    Public Class Updater
        Inherits ModuleUpdater

        Public Sub New(ByVal objectSpace As DevExpress.ExpressApp.IObjectSpace, ByVal currentDBVersion As Version)
            MyBase.New(objectSpace, currentDBVersion)
        End Sub
        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()
        End Sub
    End Class
End Namespace
